tests/phpunit/includes/class-wp-test-stream.php

```diff
@@ -57,7 +57,7 @@ private function open( $url ) {
 		$this->file   = $components['path'] ? $components['path'] : '/';
 
 		if ( empty( $this->bucket ) ) {
-			trigger_error( 'Cannot use an empty bucket name', E_USER_ERROR );
+			throw new Exception( 'Cannot use an empty bucket name' );
```

Comment:
@jrfnl should this being using `WP_Exception` instead?

Reply:
Hmm, rethinking .... stick with native PHP which most closely also aligns to these tests using PHP native `trigger_error()` instead of `wp_trigger_error()`.

Reply:
@hellofromtonya I see you already came to the same conclusion, but as an additional reason (for those coming across this PR later): these are test helpers and not tests.

Test helpers should preferably only use native PHP functionality, so as not to muddy the waters when testing - is the test code the problem or is the code under test the problem ? Using only native PHP functionality for test helpers also prevents incorrect results for code coverage recording and such.